seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json

```diff
@@ -397,7 +397,7 @@
         "The following code is an example of using <code>filter</code> to remove array elements that are equal to five:",
         "Note: We omit the second and third arguments since we only need the value",
         "<blockquote>array = array.filter(function(val) {<br>&nbsp;&nbsp;return val !== 5;<br>});</blockquote>",
-        "Use <code>filter</code> to create a new array with all the values from <code>oldArray</code> which are less than 6.<br><code>oldArray</code> should not change."
+        "Use <code>filter</code> to create a new array with all the values from <code>oldArray</code> which are less than 6.<code>oldArray</code> should not change."
```

Comment:
Can you add a space before the last sentence?

```
...less than 6. <code>oldArray</code>...
```

Make your changes and then `git add object-oriented-and-functional-programming.json; git commit --amend`. Otherwise, looks good!